scripts/cibuild

```diff
@@ -38,6 +38,7 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
 
         docker-compose \
             -f "${DIR}/../docker-compose.yml" \
+            -f "${DIR}/../docker-compose.airflow.yml" \
```

Comment:
This task doesn't seem to reference any Airflow components, so I don't think we need to include `docker-compose.airflow.yml` here.